prediction_market_agent/agents/replicate_to_omen_agent/omen_resolve_replicated.py

```diff
@@ -28,7 +28,7 @@ def omen_finalize_and_resolve_and_claim_back_all_markets_based_on_others_tx(
     private_credentials: PrivateCredentials,
 ) -> FinalizeAndResolveResult:
     public_key = private_credentials.public_key
-    balances_start = get_balances(public_key)
+    balances_start = get_balances(public_key, None)
```

Comment:
It might be worth refactoring the signature on PMAT afterwards -> (web3 : Web3|None=None), but for now this is the solution.